content/cli/orch_cli/secrets.md

```diff
@@ -32,7 +32,7 @@ Create a new secret (key-value pair)
 One of these flags:
 
 * `-s, --secret-string TEXT` - The string to use as the secret's value.
-* `-f, --secret-file TEXT` - The name of the secret file that contains the value to be set.
+* `-f, --secret-file TEXT` - The secret's file to use its content as value to be set.
```

Comment:
\- The name of the secret file that contains the value to be set.
\+ The file with the contents of the secret.